### PR TITLE
Add cancel standing/timed orders docs

### PR DESCRIPTION
**Pull request by dmytroyarmak**

### CODE REVIEW

**Review by hihuz — APPROVED** [review]
Looks good 👍

File: SDK.markdown [hunk 1]
```diff
@@ -322,6 +322,30 @@ executeAt: "<execution_date>" // e.g. 2017-03-30T12:56:54+00:00
...
+} else {
+  // wait for sms
+  const smsToken = "...";
+
+  canceledTransfer = await client.models.transfer.confirmCancelTransfer(
+    TransferType.STANDING_ORDER,
+    confirmationId,
+    smsToken
+}
+);
```
Comment (johannespfeiffer) [inline SDK.markdown]: nit: seems the two lines need to be switched.

**Review by johannespfeiffer — APPROVED** [review]
Looks good, only minor typo.

File: SDK.markdown (hunk 1 above)
Comment (dmytroyarmak) [reply SDK.markdown]: oops, fixed

**Pull request merged**